### PR TITLE
Add an option to do base64 encoding before encrypting the secret

### LINKED ISSUE #1273: Add an option to do base64 encoding before encrypting the secret
**What would you like to be added**:

For the case of Kubernetes `Secret`, the values for all keys in the data field have to be base64-encoded strings. 
So it would be nice to have an option to encode the data before encoding them.

**Why is this needed**:

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1273

**Does this PR introduce a user-facing change?**:

```release-note
Add an option to do base64 encoding before encrypting the secret
```

### CODE REVIEW

File: pkg/app/api/grpcapi/web_api.go [hunk 1]
```diff
@@ -653,8 +654,12 @@ func (a *WebAPI) GenerateApplicationSealedSecret(ctx context.Context, req *webse
 		return nil, status.Error(codes.FailedPrecondition, "The piped does not contain the encryption configuration")
 	}
 
-	var enc encrypter
+	data := req.Data
```
Comment (khanhtc1202) [inline pkg/app/api/grpcapi/web_api.go:657]: should this `data` be used on [L678](https://github.com/pipe-cd/pipe/pull/1286/files#diff-de965e19dc71ecfcaec8cde1d2deebcc289b8b1e55db8ead99e2b5e4c03ebaf0R678) 👀

Comment (nghialv) [reply pkg/app/api/grpcapi/web_api.go:657]: Nice catch! Thank you.
Fixed!

**Comment by khanhtc1202** [comment]
🚀 
/approve

**Pull request merged**